client/src/main/proto/spine/base/event.proto

```diff
@@ -73,18 +80,21 @@ message EventContext {
     CommandContext command_context = 3;
 
     // The ID of the entity which generated the event.
+    //
     // Examples of event producers are: aggregates, process managers (which generate `CommandRouted`),
     // bounded contexts (for integration events).
+    //
     google.protobuf.Any producer_id = 4;
 
     // The version of the entity after the event was applied.
-    int32 version = 5;
+    Version version = 5;
 
     // An event can be enriched with one or more messages. Or, event enrichment can be disabled.
     oneof enrichment_mode {
 
         // One or more enrichments of the event.
-        Enrichments enrichments = 6; // We are having the wrapper type because maps are not allowed in `oneof`s.
+        Enrichments enrichments = 6;
+            // We are having the wrapper type because maps are not allowed in `oneof`s.
```

Comment:
I put the comment on the next line because it 1) doesn't fit into the line 2) it's not meant to be in Javadoc of the generated class 3) I didn't want to break this sentence in two small parts.